draft-ietf-rtcweb-jsep.xml

```diff
@@ -665,6 +665,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         what it receives due to application preferences, e.g. a
         specific size for the window in which the video will be
         displayed.</t>
+
+        <t>Note that certain codecs support transmission of samples with aspect
+        ratios other than 1.0 (i.e., non-square pixels).
+        JSEP implementations will not transmit non-square pixels, but SHOULD
+        receive and render such video with the correct aspect ratio.
```

Comment:
Sorry, not sure exactly what is needed here. What did you want to see a pointer to?